linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_temp_expire_date/rule.yml

```diff
@@ -45,6 +45,7 @@ references:
     srg: SRG-OS-000123-GPOS-00064,SRG-OS-000002-GPOS-00002
     stigid@ol7: OL07-00-010271
     stigid@ol8: OL08-00-020000
+    stigid@rhel7: RHEL-07-010271
```

Comment:
The `account_emergency_expire_date` and `account_temp_expire_date` rules are basically the same with minimal changes in the rule description. It can be done in a separate PR, but we should deprecate the `account_emergency_expire_date` rule.

Reply:
That might be good idea. DISA has kept changing the wording around these a few times so that is reason for both rules.